auth0/data_source_auth0_global_client_test.go

```diff
@@ -0,0 +1,35 @@
+package auth0
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+const testAccDataGlobalClientConfig = `
+%v
+data auth0_global_client global {
+}
+`
+
+func TestAccDataGlobalClient(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		Providers: map[string]terraform.ResourceProvider{
+			"auth0": Provider(),
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(testAccDataGlobalClientConfig, testAccGlobalClientConfigWithCustomLogin),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("data.auth0_global_client.global", "custom_login_page", "<html>TEST123</html>"),
+					resource.TestCheckResourceAttr("data.auth0_global_client.global", "custom_login_page_on", "true"),
```

Comment:
I need to better understand why, but I get occasional intermittent failures with this test case.

Reply:
Still need to investigate why this acceptance test fails in the pipeline but not locally.

Reply:
Ah, because needed to actually apply the configuration before attempting to read it.